### ISSUE TITLE
Any way to overlap rectangle plot and density plot in same track?

### ISSUE THREAD

**Issue by rpugho** [jokergoo/circlize#366]
Want to overlap density plot and rectangle plot in the same track, like this :
![Untitled](https://user-images.githubusercontent.com/66585204/219922092-9c9310c3-f2df-43b4-8118-c0b441c34834.png)
tried to use circos.genomicTrack and panel.fun but seems they are not compatible

**Comment by jokergoo** [comment]
Yes, density plots are polygons and you can manually draw these graphics with `circos.rect()` and `circos.polygon()`.

Maybe you can provide me some test datasets?
